### PR TITLE
some more work on debugging issues in event streams

### PR DESCRIPTION
**Pull request by whyrusleeping**

### CODE REVIEW

File: api/atproto/syncsubscribeRepos.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ func init() {
 
 type SyncSubscribeRepos_Commit struct {
 	Blobs  []util.LexLink               `json:"blobs" cborgen:"blobs"`
-	Blocks util.LexBytes                `json:"blocks" cborgen:"blocks"`
+	Blocks util.LexBytes                `json:"blocks,omitempty" cborgen:"blocks"`
```
Comment (bnewbold) [inline api/atproto/syncsubscribeRepos.go:14]: this is a manual hack, and would get clobbered by running `lexgen`, I assume? Ok for now, making lexgen "clean" is a goal but will be a separate pass of work.

File: cmd/gosky/debug.go [hunk 2]
```diff
@@ -231,3 +242,152 @@ var debugStreamCmd = &cli.Command{
 		return nil
 	},
 }
+
+// TODO: WIP - turns out to be more complicated than i initially thought
```
Comment (bnewbold) [inline cmd/gosky/debug.go:246]: feels like this is almost a separate tool, though don't want to proliferate those too much. maybe do move code for this specific command to a separate file at least? it is pretty big

**Review by bnewbold — APPROVED** [review]
looks like the second commit is mostly just detecting the fork condition, but not doing anything yet, right? seems ok. we are really starting to pile up TODOs to handle in the next ~week.

File: api/atproto/syncsubscribeRepos.go (hunk 1 above)
Comment (whyrusleeping) [reply api/atproto/syncsubscribeRepos.go:14]: Yeah, lexgen needs work. I guess i can add a TODO comment here to help us not miss it in future diffs

**Pull request merged**